.eslintrc.json

```diff
@@ -3,7 +3,9 @@
     "plugins": [
         "@typescript-eslint",
         "header",
-        "deprecation"
+        "deprecation",
+        "simple-import-sort",
+        "import"
```

Comment:
You mentioned in the Description that you liked simple-import-sort the best. Were we not able to get everything we need from that package? Is that why we also need this second `import` library?

Reply:
`simple-import-sort` recommended the second plugin in their [example config](https://github.com/lydell/eslint-plugin-simple-import-sort/#example-configuration), so I still count it as a part of option 4. And yeah it added some extra stuff. For example, the `import` plugin will combine these into one:

```
import { EventGridFunctionOptions } from '@azure/functions';
import { CosmosDBFunctionOptions } from '@azure/functions';
```